cmd/controller-manager/app/controller-manager.go

```diff
@@ -301,9 +301,23 @@ func updateFederationConfig(config *rest.Config, fedConfig *corev1a1.FederationC
 func setOptionsByFederationConfig(opts *options.Options) {
 	fedConfig := getFederationConfig(opts)
 	if fedConfig == nil {
+		// Computing the right value of the namespace based on how
```

Comment:
This logic is duplicated from the `Run` method and serves no purpose here.  Please remove.

Reply:
done!